dist/rpm/phoebe.spec

```diff
@@ -54,7 +51,7 @@ to offer eventually the best possible setup.
 %build
 # export build flags manually if %%set_build_flags is not defined
 %{?!set_build_flags:export CFLAGS="%{optflags}"; export LDFLAGS="${RPM_LD_FLAGS}"}
-%meson -Dwith_tests=false -Dsystemd_system_unit_dir=%{_unitdir}
+%meson -Dwith_tests=false -Dsystemd_system_unit_dir=%{_unitdir} -Dcollector=disabled
```

Comment:
Adding this option makes no difference for the resulting rpm (it's just faster to build and has less build time dependencies), so I'm in favor of this.